Codigos/Estruturas-de-Dados/XorTrie/XorTrie.cpp

```diff
@@ -0,0 +1,63 @@
+template <int bits = 30> struct XorTrie {
+    vector<vector<int>> go;
+    int root, cnt;
+
+    int new_node() {
+        go[0][cnt] = -1;
+        go[1][cnt] = -1;
```

Comment:
Better to reverse the order and can be one-lined
```suggestion
        go[cnt][0] = go[cnt][1] = -1;
```